config/routes.rb

```diff
@@ -2,7 +2,8 @@
   ActiveAdmin.routes(self)
   root :to => 'home#index'
   get '/error' => 'home#error'
-  get '/me'    => 'home#me'
+  get '/me'    => 'profile#show'
```

Comment:
Sorry, not sure what you mean by "regular"? How would you construct the routes differently?